lib/codegen/model.ejs

```diff
@@ -24,20 +24,29 @@ function printMethod(modelName) {
     return method + params.join(', ') + ') {';
 }
 
+function joinParams(params) {
```

Comment:
this method is call when code is being generated by SoapGenerator, and the parameters are all string names (impossible to see if parameter object has toJSON() method.